Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -1559,9 +1560,8 @@ void CFrame::ConnectWiimote(int wm_idx, bool connect)
 	{
 		bool was_unpaused = Core::PauseAndLock(true);
 		GetUsbPointer()->AccessWiiMote(wm_idx | 0x100)->Activate(connect);
-		wxString msg(wxString::Format(_("Wiimote %i %s"), wm_idx + 1,
-					connect ? _("Connected") : _("Disconnected")));
-		Core::DisplayMessage(WxStrToStr(msg), 3000);
+		const char* message = connect ? "Wiimote %i connected" : "Wiimote %i disconnected";
```

Comment:
It's the same reason as for every other call to DisplayMessage. Should I add a comment just because this one is in DolphinWX?